recipes/leveldb/all/conandata.yml

```diff
@@ -0,0 +1,7 @@
+sources:
+  "1.21":
```

Comment:
Is `1.21` required or does `1.22` suffice?

Reply:
not sure - is it recommended to start new recipes from the latest known version at the time and retain them for some period? I can switch this to 1.22 right now and revisit later if there is a need for 1.21. There are a couple of other libraries that I intend to submit follow-up recipes for that currently use leveldb 1.20 (which unfortunately does not use cmake), but I'll cross that bridge when i get there.

Reply:
Older versions are kept because somebody might depend on it,
but this is a new recipe/version so nobody can depend on it.
It's up to you, the author, to make the decision.